### DISCUSSION TITLE
Segnalazioni in scadenza cancellate

### DISCUSSION THREAD

**Discussion by Cangio84** [globaleaks/globaleaks-whistleblowing-software#3930] [Q&A]
Buongiorno, gli utenti che gestiscono le segnalazioni hanno ricevuta questa comunicazione via email:

_Gentile utente
Una o più segnalazioni sono in scadenza e verranno presto cancellate.
La scadenza più vicina è Monday 01 January 2024 00:00 (UTC).
Ricordati di controllarle prima che vengano cancellate.
https://whistleblowing.xxxxxx.it/#/login
Distinti saluti,
Whistleblowing_

Da regolamento le segnalazioni non posso essere cancellate in automatico. C'è qualche opzione che forse non attivato?
è corretto che le segnalazioni, se non gestite, vengano cancellate?
In questo momento non ho possibilità di verificare se ci sono delle segnalazioni in scadenza non gestite, ho il ruolo di admin (tenico)

Grazie,

**Comment by evilaliv3** [comment]
Grazie per la domanda.

Secondo GDPR è importante che l'ente strutturi un processo in cui si assicuri che ogni informazione ricevuta abbia una chiara e definita policy di data retention.
GlobaLeaks è progettato per supportare l'ente anche con policy automatizzate in piena compliance con la normativa vigente ma se ciò non si confà alle procedure interne dell'ente che preferisce operare completamente in modo manuale questo è ottenibile riconfigurando il sistema e impostando le policy che si ritengono opportune.

**Reply by Cangio84** [reply]
Perdona ma non è molto chiara la risposta. Chi deve decidere quanto devono essere conservate le segnalazioni?
Quindi anche delle segnalazioni gestite e chiuse, vengono cancellate?
Non ho modo di recuperare delle segnalazioni eliminate da sistema?
Grazie,
Alessandro

**Comment by evilaliv3** [comment]
A decidere quanto tenere le segnalazioni è il ricevente incaricato che tipicamente procede secondo policy aziendale e secondo legge.

GlobaLeaks per supportare un processo di gestione secondo GDPR (che chiede di cancellare ciò che non è necessario) di default cancella indelebilmente ogni segnalazione dopo 90 giorni.
E' compito dell'amministrazione definire una policy di default utile agli utenti/clienti eventualmente anche valutando di disabilitare completamente questa policy.

Non c'è alcun modo di recuperare una segnalazione cancellata in quanto la cancellazione è volutamente progettata per essere sicura.

**Comment by Cangio84** [comment]
Grazie evilaliv3.
io gestisco la parte tecnica e volevo sapere, dal punto di vista tecnico, se lato portale c'è qualche impostazione che permette di definire quanto tempo tenere le segnalazioni.
Nel dettaglio le colleghe si sono accorte che la segnalazione era stata aperta il 02/10/23, non si ricordano quando è stata chiusa ed il 01/01/2024 è stata eliminata dopo i vari avvisi via email.
Oggi andranno ad impostare come "chiusa" un'altra segnalazione aperta il 23/11/2023. dopo 90 giorni verrà eliminata? posso decidere io quanto tenerla dopo la chiusura.
Grazie ancora

**Comment by evilaliv3** [comment]
Per ogni canale di segnalazione trovate una opzione "Scadenza" in cui impostare dopo quanti giorni fare scadere una segnalazione.
Potete ad esempio indicare 365 per impostare che una segnalazione venga cancellata automaticamente dopo un anno dall'arrivo.
Parimenti potete impostare 0 per disabilitare questa policy automatica e in questo caso dovrete dare agli utenti il permesso di cancellare le segnalazioni altrimenti vi ritrovereste a conservarle tutte indefinitamente.

**Comment by Cangio84** [comment]
Perfetto. chiarissimo e gentilissimo.
